Documentation/install/system_requirements.rst

```diff
@@ -289,6 +289,19 @@ ICMP 8/0                 egress          ``worker-sg`` (self) health checks
           Routing mode, you can condense all rules into ingress/egress ANY
           port/protocol to/from self.
 
+The following ports should also be available on each node:
+
+======================== ==========  ==========================================
+Port Range / Protocol    Interface   Description
+======================== ==========  ==========================================
+4240/tcp                 pod IP      cluster health checks (``cilium-health``)
+4244/tcp                 pod IP      hubble server
+4245/tcp                 all         hubble relay
+6942/tcp                 pod IP      operator Prometheus metrics
+9090/tcp                 pod IP      cilium-agent Prometheus metrics
```

Comment:
I'm actually not sure what `pod IP` means in this context. Cilium runs in host networking mode, and AFAIK we typically bind to the ports on all devices on the node. One possible exception would be `cilium-operator`, I didn't check whether that runs in host networking mode.